src/components/FormSchemaEditor/FormQuestionSchemaEditor.tsx

```diff
@@ -0,0 +1,237 @@
+  });
+
+  const paragraphPreviewStyles = classNames({
+    "w-full px-2 py-1 rounded-md placeholder-secondary border-2 border-inactive resize-none box-border text-secondary bg-tertiary \
+    focus:ring-2 focus:ring-primary focus:ring-opacity-20 focus:outline-none pointer-events-none":
+      true,
+    "bg-tertiary": hovered,
+  });
+
+  const questionPreview = () => {
+    switch (question.type) {
+      case "short-answer":
+        return (
+          <input
+            readOnly
+            value="Short Answer"
+            className={singlelinePreviewStyles}
+            onClick={(e) => {
+              e.stopPropagation();
+            }}
+          />
+        );
+      case "long-answer":
+        return (
+          <textarea
+            readOnly
+            value="Long Answer"
+            className={paragraphPreviewStyles}
+            onClick={(e) => {
+              e.stopPropagation();
+            }}
+          />
+        );
+      // case "multiple-choice":
+      //   return (
+      //     <div>
+      //       {[...question.choices].map((choice) => {
+      //         return (
+      //           <label>
+      //             <input type="radio" name={choice} value="6/24" id={choice} />
+      //             {choice}
+      //           </label>
+      //         );
+      //       })}
+      //     </div>
+      //   );
+      default:
+        return <div></div>;
+    }
+  };
+
+  const wrapperStyles = classNames({
+    "bg-white w-full p-6 rounded border-2 border-box": true,
+    "border-white": !focused,
+    "border-secondary": focused,
+  });
+
+  return (
+    <Draggable key={question.id} draggableId={question.id} index={index}>
+      {(provided, _) => {
+        return (
+          <div
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            style={{
+              // some basic styles to make the items look a bit nicer
+              userSelect: "none",
+
+              // change background colour if dragging
+              // background: snapshot.isDragging ? "lightgreen" : "white",
+
+              // styles we need to apply on draggables
+              ...provided.draggableProps.style,
+            }}
+          >
+            <div
+              ref={ref}
+              onMouseDown={() => {
+                setFocused(true);
+              }}
+              onMouseEnter={() => {
+                setHovered(true);
+              }}
+              onMouseLeave={() => {
+                setHovered(false);
+              }}
+              className={wrapperStyles}
+            >
+              <div className="flex items-center justify-between overflow-hidden">
+                <DragHandle
+                  className="cursor-grab p-1.5"
+                  {...provided.dragHandleProps}
+                />
+                <div className="w-2"></div>
+
+                <div className="w-1"></div>
+                <div className="rounded hover:bg-tertiary p-1.5 cursor-pointer">
```

Comment:
Pressing on the outer part of the delete icon (hover background) does not initiate onClick function which is confusing

Reply:
Nice catch--thanks!